great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -246,7 +244,7 @@ def _build_parameters(
         )
 
         return (
-            regex_string_success_list,
+            best_regex_string,
```

Comment:
@Shinnnyshinshin We are almost there.  The only thing left to provide is the `success_ratio` -- I do not see it in the `details` dictionary.  If we look at the `SimpleDateFormatStringParameterBuilder`, we see:
```
        return (
            best_fmt_string,
            {
                "success_ratio": best_ratio,
                "candidate_strings": sorted(candidate_strings),
            },
        )
```

Thanks.

Reply:
@alexsherstinsky in line with our earlier discussion, the output is: 
```
{
  evaluated_regexes {
    s1: s1_success_ratio
    s2: s2_success_ratio
    s3: s3_success_ratio
  }
  threshold: threshold
}
```

Reply:
@Shinnnyshinshin Like we discussed, we do not want `threshold` -- we want the best `success_ratio`.  Thanks.